todo_app/data/session_items.py

```diff
@@ -67,3 +67,20 @@ def save_item(item):
     session['items'] = updated_items
 
     return item
+
+def remove_item(id):
+    """
+    Removes an existing item in the session. If no existing item matches the ID of the specified item, nothing is removed.
+
+    Args:
+        item: The item to save.
+    """
+    existing_items = get_items()
+    for i in range(len(existing_items)):
+        if existing_items[i]['id'] == int(id):
+            del existing_items[i]
+            break
+
+    session['items'] = existing_items
+
+    return
```

Comment:
You don't need an empty return statement if it's the end of the function anyway. You only write this if you want to return early. E.g.
```
    existing_items = get_items()
    for i in range(len(existing_items)):
        if existing_items[i]['id'] == int(id):
            del existing_items[i]
            session['items'] = existing_items
            return
```